sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsub/PubsubIO.java

```diff
@@ -856,7 +856,8 @@ private PubsubIO() {}
   /** Implementation of {@link #write}. */
   @AutoValue
   public abstract static class Write<T> extends PTransform<PCollection<T>, PDone> {
-    private static final int MAX_PUBLISH_BATCH_BYTE_SIZE_DEFAULT = 10 * 1024 * 1024;
+    /** Max batch byte size takes into account size increase after encoding messages. */
+    private static final int MAX_PUBLISH_BATCH_BYTE_SIZE_DEFAULT = 7864320;
```

Comment:
Can we replace const with a formula and reference base64 in the comment? THank you.

Reply:
@tvalentyn Done